packages/lab/src/components/Wizard/WizardContext/WizardContext.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {
+  createContext,
+  useState,
+  Dispatch,
+  SetStateAction,
+  useCallback,
+} from "react";
+
+export type ContextProp = {
```

Comment:
You may add a generic type to extend more properties for each tab like I proposed on [this line](https://github.com/lumada-design/hv-uikit-react/pull/3297/files#diff-1ef21b5620f3174f5bd0e8891d621e118da7505aa9b995cb17b2d953b36c8e9cR52).